README.md

```diff
@@ -0,0 +1,23 @@
+# BeGUI
+BeGUI is a simple to use programmer's tool which allows for rapid development
+of a program.
+
+![BeGUI prefs screenshot](begui.jpg)
+
+
+Building
+--------
+
+Simply go into BeGUI directory and type ``bash creationx86.script``.
+
+
+Copyright
+------------
+
+Original 1999 version by Ed Musgrove, enhanced and updated for Haiku in 2018 by Arnav Bhatt.
+
+
+Contact
```

Comment:
You can drop the contact info.  Since it's on HaikuArchives users can use the bug tracker here to report issues.  You say it doesn't really work on Haiku, but I see zero open issues.  Can you do some testing of this and report what you find.  Does it build without errors, does it launch, do menu items work, etc.  Open a new issue for things you discover.

Reply:
Alright, I will give you bug report and open issue.